drivers/gles2/shaders/canvas.glsl

```diff
@@ -310,20 +310,20 @@ VERTEX_SHADER_CODE
 // Do not copy these defines in the vertex section.
 #ifndef USE_GLES_OVER_GL
 #ifdef GL_EXT_shader_texture_lod
-#extension GL_EXT_shader_texture_lod : enable
-#define texture2DLod(img, coord, lod) texture2DLodEXT(img, coord, lod)
-#define textureCubeLod(img, coord, lod) textureCubeLodEXT(img, coord, lod)
+//#extension GL_EXT_shader_texture_lod : enable
```

Comment:
I'm concerned about this - are the extensions "supported" but not working on Vita?

Reply:
This is exactly the case

Reply:
Commented code is always very concerning. This is a question for @SonicMastr.

Reply:
> There's a better way to handle this

@SonicMastr Besides preprocessors, what were you thinking of?